lib/index.js

```diff
@@ -82,6 +84,14 @@ function vfileMessageToDiagnostic(message) {
     }
   }
 
+  if (
+    typeof message.cause === 'object' &&
+    message.cause &&
+    'stack' in message.cause
+  ) {
+    diagnostic.message += '\n' + message.cause.stack
+  }
```

Comment:
You might want to support causes of causes: https://github.com/vfile/vfile-reporter/commit/40c1a1d5ce694d7fba091e0be1f0841e355bb9e8

Reply:
Also, not all errors have a `stack` field

Reply:
I handled causes using `util.inspect`. Typically `util.inspect` shouldn’t be used for production code, bu I  think it’s ok here, because we just want to display whatever the cause is.

Reply:
Probably fine, yeah, we’re also using `node:fs` (in `find-up` I presume)